kolibri/plugins/audio_mp3_render/assets/src/vue/index.vue

```diff
@@ -56,7 +56,7 @@
       isPlay: true,
       isPause: false,
       max: 0,
-      // This data attribute is required, as we cannot use this.$els.audio in our getter for
+      // This data attribute is required, as we cannot use this.$refs.audio in our getter for
       // rawTime, because at the time of getter initialization for the computed property,
       // the DOM does not exist, so the above object path is undefined, which causes problems.
```

Comment:
we may want to check whether this comment is still true, or if there are other work-arounds post Vue2. I'll ad a new card to track these kinds of items